forum/qa-plugin/q2a-change-username-limit/page/account.php

```diff
@@ -0,0 +1,548 @@
+        ),
+
+        'messages' => array(
+            'label' => qa_lang_html('users/private_messages'),
+            'tags' => 'name="messages"',
+            'type' => 'checkbox',
+            'value' => !($useraccount['flags'] & QA_USER_FLAGS_NO_MESSAGES),
+            'note' => qa_lang_html('users/private_messages_explanation'),
+        ),
+
+        'messages_email' => array(
+            'label' => qa_lang_html('users/private_messages_email'),
+            'tags' => 'name="messages_email"',
+            'type' => 'checkbox',
+            'value' => $useraccount['pwemail'],
+            'note' => qa_lang_html('users/private_messages_email_explanation'),
+        ),
+
+        'activity_email' => array(
+            'label' => qa_lang_html('users/activity_email_notifications'),
+            'tags' => 'name="activity_email"',
+            'type' => 'checkbox',
+            'value' => $useraccount['activityemail'],
+            'note' => qa_lang_html('users/activity_email_notifications_explanation'),
+        ),
+
+        'wall' => array(
+            'label' => qa_lang_html('users/wall_posts'),
+            'tags' => 'name="wall"',
+            'type' => 'checkbox',
+            'value' => !($useraccount['flags'] & QA_USER_FLAGS_NO_WALL_POSTS),
+            'note' => qa_lang_html('users/wall_posts_explanation'),
+        ),
+
+        'mailings' => array(
+            'label' => qa_lang_html('users/mass_mailings'),
+            'tags' => 'name="mailings"',
+            'type' => 'checkbox',
+            'value' => !($useraccount['flags'] & QA_USER_FLAGS_NO_MAILINGS),
+            'note' => qa_lang_html('users/mass_mailings_explanation'),
+        ),
+
+        'avatar' => null, // for positioning
+    ),
+
+    'buttons' => array(
+        'save' => array(
+            'tags' => 'onclick="qa_show_waiting_after(this, false);"',
+            'label' => qa_lang_html('users/save_profile'),
+        ),
+    ),
+
+    'hidden' => array(
+        'dosaveprofile' => '1',
+        'code' => qa_get_form_security_code('account'),
+    ),
+);
+
+if (qa_get_state()=='profile-saved') {
+    $qa_content['form_profile']['ok'] = qa_lang_html('users/profile_saved');
+}
+
+if (!qa_opt('allow_private_messages')) {
+    unset($qa_content['form_profile']['fields']['messages']);
+}
+
+if (!qa_opt('allow_user_walls')) {
+    unset($qa_content['form_profile']['fields']['wall']);
+}
+
+if (!qa_opt('mailing_enabled')) {
+    unset($qa_content['form_profile']['fields']['mailings']);
+}
+
+if ($isblocked) {
+    unset($qa_content['form_profile']['buttons']['save']);
+    $qa_content['error']=qa_lang_html('users/no_permission');
+}
+
+//	Avatar upload stuff
+
+if (qa_opt('avatar_allow_gravatar') || qa_opt('avatar_allow_upload')) {
+    $avataroptions=array();
+
+    if (qa_opt('avatar_default_show') && strlen(qa_opt('avatar_default_blobid'))) {
+        $avataroptions['']='<span style="margin:2px 0; display:inline-block;">'.
```

Comment:
Dlaczego styl inline?

Reply:
To jest kopia pliku oryginalnego. Zaznaczyłem miejsca w których dokonałem zmiany.